.vscode/extensions.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
note: Something is wrong in this PR as it has commits from the other PR https://github.com/cratera/papers-game/pull/20 ;)

Reply:
Ah it has the wrong base lol, I'll fix it

Reply:
Since they are all based on each other, as soon as I merge a PR, the nex one points to `revamp-2023` automatically but they still have the prevous PR as base 😄